sfft_and_animate.py

```diff
@@ -177,12 +168,35 @@ def sfft_and_animate(oid,band):
         ref_4k_savepath = f'/work/lna18/imsub_out/coadd/cutouts_4k/{oid}_{band}_{pointing}_{sca}_coadd_4kcutout.fits'
         ref_4k = stampmaker(RA,DEC,coadd_img_path,shape=np.array([4088,4088]),savepath=ref_4k_savepath)
 
-        sci_skysub_path = sky_subtract(band=band,pointing=pointing,sca=sca)
-        sci_imalign_path = imalign(ref_4k,sci_skysub_path,force=True)
-
         print('REF_4K_PATH')
         print(ref_4k)
 
+        # Retrieve and rotate PSFs for PSF coaddition.
+        template_psf_list = []
+        for i, row in enumerate(out_tab):
+            psf = get_imsim_psf(RA,DEC,out_tab['filter'][i],out_tab['pointing'][i],
```

Comment:
enumerating and then using the index seems a little odd.  I think you want something simpler like:
```
for row in out_tab:
    psf = get_imsim_psf(RA, Dec, row["filter"], row["pointing"], row["sca"])
    template_psf_list.append(psf)
```

Reply:
Oops, this is an artifact of a brain fart. Thanks for catching.